README.md

```diff
@@ -118,4 +118,12 @@ $ heroku container:release web -a mongoappex10
 ```
 ## migration from trello API to MongoDB
 Update environment variable [MONGO_USERNAME] ,[MONGO_PASSWORD],[MONGO_DB],[MONGO_URL] for the MongoDb connection in the .env file
-These parameter should be included for Heroku and Travis-CI environment variables
\ No newline at end of file
+These parameter should be included for Heroku and Travis-CI environment variables
+
+## GitHub Authentication and roles authorisation
+The application requires the Github client-id and client-secret and redirect Url to set as environment variables -
+[GITHUB_CLIENT_ID] ,[GITHUB_CLIENT_SECRET],[GITHUB_REDIRECT_URI]
```

Comment:
It would be nice to add these to your .env.template like you have with the Mongo variables